__tests__/__unit__/extension.unit.test.ts

```diff
@@ -1375,907 +1494,769 @@ describe("Extension Unit Tests", () => {
-
-        showInputBox.mockReturnValueOnce("test");
-        await dsActions.enterPattern(node, testTree);
+    it("Testing that openRecentMemberPrompt (opening a recent member) is executed successfully on a USS file", async () => {
+        const sessNode2 = new ZoweUSSNode("sessNode2", vscode.TreeItemCollapsibleState.Expanded, null, session, "", false, "testProf");
+        sessNode2.contextValue = globals.DS_SESSION_CONTEXT;
+        sessNode2.fullPath = "";
+        const node = new ZoweUSSNode("node3.txt", vscode.TreeItemCollapsibleState.None, sessNode2, null, "/node1/node2");
+        node.contextValue = globals.DS_DS_CONTEXT;
+        sessNode2.children.push(node);
+        testUSSTree.mSessionNodes.push(sessNode2);
 
-        expect(showInputBox.mock.calls.length).toBe(1);
-        expect(showInputBox.mock.calls[0][0]).toEqual({
-            prompt: "Search data sets by entering patterns: use a comma to separate multiple patterns",
-            value: node.pattern
+        const qpItem: vscode.QuickPickItem = new utils.FilterDescriptor("\uFF0B " + "Create a new filter");
+        const resolveQuickPickHelper = jest.spyOn(utils, "resolveQuickPickHelper").mockImplementation(
+            () => Promise.resolve(qpItem)
+        );
+        createQuickPick.mockReturnValue({
+            activeItems: [node.label],
+            ignoreFocusOut: true,
+            items: [node.label],
+            value: "[testProf]: /node1/node2/node3.txt",
+            show: jest.fn(() => {
+                return {};
+            }),
+            hide: jest.fn(() => {
+                return {};
+            }),
+            onDidAccept: jest.fn(() => {
+                return {};
+            })
         });
-        expect(showInformationMessage.mock.calls.length).toBe(0);
 
-        showInputBox.mockReturnValueOnce("");
+        showQuickPick.mockReset();
         showInputBox.mockReset();
-        showInformationMessage.mockReset();
-        await dsActions.enterPattern(node, testTree);
 
-        expect(showInformationMessage.mock.calls.length).toBe(1);
-        expect(showInformationMessage.mock.calls[0][0]).toBe("You must enter a pattern.");
+        mockGetRecall.mockReturnValueOnce([]);
+        mockUSSGetRecall.mockReturnValueOnce([`[testProf]: /node1/node2/node3.txt`]);
+
+        await sharedActions.openRecentMemberPrompt(testTree, testUSSTree);
+        expect(testUSSTree.openItemFromPath).toBeCalledWith(`/node1/node2/node3.txt`, sessNode2);
+
+        testTree.mSessionNodes.pop();
+        showQuickPick.mockReset();
+        showInputBox.mockReset();
     });
 
-    it("Testing that enterPattern is executed successfully for search favorite", async () => {
-        mockAddZoweSession.mockReset();
-        const favoriteSample = new ZoweDatasetNode("[sestest]: HLQ.TEST", vscode.TreeItemCollapsibleState.None, undefined, null);
+    it("Testing that that openPS credentials prompt is executed successfully", async () => {
```

Comment:
All credential checks have been covered in `profiles.unit.test.ts` so they can be removed